doc/db.md

```diff
@@ -42,6 +42,30 @@ part of [Qcode Documentation](index.md)
 * [sql_where_in]
 * [sql_where_in_not]
 
+### Database inspection
```

Comment:
The term "introspection" seems to be used more readily in this context.
I did a Google search for "database inspection" vs "database introspection"